extensions/BMO/template/en/default/pages/anti-abuse.html.tmpl

```diff
@@ -0,0 +1,59 @@
+<!-- 1.0@bugzilla.org -->
+[%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  #
+  # This Source Code Form is "Incompatible With Secondary Licenses", as
+  # defined by the Mozilla Public License, v. 2.0.
+  #%]
+
+[% PROCESS global/header.html.tmpl 
+   title = "Bugzilla Anti-Abuse Policy" 
+   style = "li { margin: 5px } .heading { font-weight: bold }" %]
+
+<h2>Anti-Abuse Policy</h2>
+
+<p>
+  It is our intention that [% terms.Bugzilla %] remains a useful tool for reporting 
+  and commenting on [% terms.bugs %], feature-requests and tasks for the Mozilla community.
+</p>
+
+<p>
+  In order to keep [% terms.Bugzilla %] a useful, inclusive place for this work, we have 
+  <a href="page.cgi?id=etiquette.html">guidelines</a> 
+  for using this site which, by using this site, you agree to follow.
+</p>
+
+<p>
+  If you find a [% terms.Bugzilla %] user violating or <a href="page.cgi?id=etiquette.html">guidelines</a> 
+  or  <a href="https://www.mozilla.org/about/governance/policies/participation/">Mozilla's participation guidelines</a> 
+  in comments on [% terms.bugs %], please <a href="https://wiki.mozilla.org/BMO/comment_tagging">tag the comments</a> 
+  in question so that an administrator can review them. 
+<p>
+
+<p>
+  If a comment is abusive or threatening use the tag <b><i>admin</i></b>. An admin will 
+  receive a notification shortly and be able to follow up. [% terms.Bugs %] with comments marked 'spam',
+  and 'advocacy' will also be reviewed.
+</p>
+
+<p>
+  If a [% terms.bug %]'s short-description or whiteboard tags violate our guidelines,
+  please describe the issue in a comment and tag it <b><i>admin</i></b>.
+</p>
+
+<p>
+  If another [% terms.Bugzilla %] user asks you to stop a behavior, stop doing it. 
+  If an administrator asks you to stop engaging in a behavior, do it. Administrators 
+  have the discretion to limit or suspend your account, and take further actions 
+  if necessary if you violate our guidelines.
+</p>
```

Comment:
It's not clear how this document fits in with the Etiquette document. Until this paragraph, it seemed to be that the Etiquette document said what to do or not do, and this document explained what to do when you find a violation. (Which would mean it should be titled the Abuse Process, not the Anti-Abuse Policy.) But then we get to this paragraph, which seems duplicative with the Etiquette document.

I think we need to think through their proper relationship, and either merge them, or clearly articulate the difference and update this document to fulfil only the function which it's supposed to have which is different from the Etiquette document.